### ISSUE TITLE
Private github repositories are not listed in Github repo selection step

### ISSUE THREAD

**Issue by rav94** [mike-goodwin/owasp-threat-dragon#119]
The repository was listed only after I changed the repository permissions to the public mode. Think this would be an important option to have because you are providing the Github credentials expecting to view your private repos as well. Plus people would be reluctant to put their threat model diagrams in public.

**Comment by MRAK96** [comment]
+1
Any plan to add support for private repo?
We cant let our threat diagrams visible to public eyes.

**Comment by jgadsden** [comment]
Hello @rav94 and @infa-aksharma , agreed that this is a very good point. We do not have anyone working on this at present, and not sure when this will change.
Are you able to look at this issue yourself and propose a solution? Or if you know of anyone else that could contribute, this would be great.

The alternative is to use Threat Dragon desktop version on a locally cloned repo, and then commit+push back up to the private repo when required. I know this is not using the webapp, and is more awkward process, but it may help here.

The desktop release is at :
https://github.com/mike-goodwin/owasp-threat-dragon-desktop/releases/tag/v1.2

**Comment by jgadsden** [comment]
Migrated to new issue in the OWASP area repo : https://github.com/OWASP/threat-dragon/issues/8
